### PR TITLE
Fix navigateTo description

### PR DESCRIPTION
**Pull request by kasarchou**
Fixes appsmithorg/appsmith#9564

### CODE REVIEW

**Comment by kasarchou** [comment]
Fixes appsmithorg/appsmithorg#9564

File: framework-reference/navigateto.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 description: >-
   With navigateTo, we can navigate in between the internal pages of Appsmith. It
```
Comment (Pranay105) [inline framework-reference/navigateto.md:3]: navigateTo allows the user to navigate between the internal pages of the App or to an external URL.

File: framework-reference/navigateto.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
...
   could be triggered on any widget action like Button onClick, Dropdown
-  onOptionChange etc. Create a new Page with a valid
+  onOptionChange etc. Create a new Page with a valid name or use a URL from an
+  external webpage. Then assign it to navigateTo to be redirected to it when the
```
Comment (Pranay105) [inline framework-reference/navigateto.md:6]: Enter the page name or the external URL in the navigateTo function (under a triggerable action like onClick), enter the Query parameters if required, and select the destination for the new page (a new window or same window.)

**Review by Pranay105 — CHANGES_REQUESTED** [review]
Great work @kasarchou . I just suggested some structural changes, Please have a look.

**Comment by kasarchou** [comment]
I committed the changes you requested. You are right that explains the method far better. Thank you for your help @Pranay105 .
Although i hope i committed it  correctly, if not please let me know.

**Review by Pranay105 — APPROVED** [review]

**Comment by Rishabhkaul** [comment]
@Pranay105 The changes you had proposed don't seem to be reflecting. The intro paragraph still ends with a random sentence "Enter t"
Screenshot 2022-03-14 at 3 10 03 PM

**Comment by Pranay105** [comment]
This happened because the text came under the page name description. I think one can add limited text here. It should come below. @Rishabhkaul, I'll fix it.

**Pull request merged**